clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -155,6 +155,29 @@ public ConfigDef define(String name, Type type, Object defaultValue, Validator v
         return define(new ConfigKey(name, type, defaultValue, validator, importance, documentation, group, orderInGroup, width, displayName, dependents, recommender, false));
     }
 
+    /**
+     * Define a new configuration
+     * @param name              the name of the config parameter
+     * @param type              the type of the config
+     * @param defaultValue      the default value to use if this config isn't present
+     * @param validator         the validator to use in checking the correctness of the config
+     * @param importance        the importance of this config
+     * @param documentation     the documentation string for the config
+     * @param group             the group this config belongs to
+     * @param orderInGroup      the order of this config in the group
+     * @param width             the width of the config
+     * @param displayName       the name suitable for display
+     * @param dependents        the configurations that are dependents of this configuration
+     * @param recommender       the recommender provides valid values given the parent configuration values
+     * @param overwrittenValue  the overwritten value of this configuration
```

Comment:
> the overwritten value

Not every descriptive IMHO. Can we explain it better?

Reply:
Sure. I've changed it to sth more descriptive